### DISCUSSION TITLE
Add placeholder

### DISCUSSION THREAD

**Discussion by djnejk** [Zrips/CMI#3565] [Feature Requests]
** Description of the improvement request: **
Hello,

I have an idea to add a placeholder "%no_vanished_online%" this placeholder would show the number of online players, but would not count the vanished players.

---
### Basic explanation of why this feature is needed and / or useful for obtaining: ###
This feature is useful so that, for example, the number of online players can be seen in the tab, except for vanished players

**Comment by CloudeLecaw** [comment]
this is normal allready in cmi, vanished players dont count now even in the server online amount

**Comment by djnejk** [comment]
I would like to ask through which placeholder this can be viewed. (Using the PlaceHolderAPI plugin)
Using placeholder "%server_online%"

I have the plugin updated to the latest version, but in TAB it looks pretty weird
A player sees only one name (his name), but the number of online players is 2

<img width="623" alt="tab" src="https://user-images.githubusercontent.com/61634734/86647368-476b1100-bfe0-11ea-8fcd-3b0fbb1ec9db.PNG">
